### DISCUSSION TITLE
Please add sort by date added to the Rules window (to help easily find most recent rules added)

### DISCUSSION THREAD

**Discussion by m-git-h** [objective-see/LuLu#596] [Ideas]
Any chance you can add  "date added" and allow us to sort by date added to the rules window?  

This would help tremendously find new rules you just denied or approved.   

Right now it is hard to find a new rule you just approved or disapproved if you forgot what application popped up.

**Comment by objective-see** [comment]
Thanks for the request! 

In the latest (v3.0+) versions, I've added a new view (`Recent Rules`) that shows the most recent rules (from the last 24 hours), sorted by their creation time:
![image](https://github.com/user-attachments/assets/9eba5e2b-3338-4d3e-9c9c-f6a0f5e86fa7)

Hope that helps!
